osu.Game/Beatmaps/BeatmapDifficulty.cs

```diff
@@ -20,7 +20,7 @@ public class BeatmapDifficulty : EmbeddedObject, IBeatmapDifficultyInfo
         public float OverallDifficulty { get; set; } = IBeatmapDifficultyInfo.DEFAULT_DIFFICULTY;
         public float ApproachRate { get; set; } = IBeatmapDifficultyInfo.DEFAULT_DIFFICULTY;
 
-        public double SliderMultiplier { get; set; } = 1;
+        public double BaseSliderVelocity { get; set; } = 1;
```

Comment:
As mentioned [on discord](https://discord.com/channels/188630481301012481/188630652340404224/1105201325647614062), this rename makes the entire pull unreviewable. It causes the local database to break down completely because this is persisted to realm and a migration is necessary. Or a removal of the value from realm, which https://github.com/ppy/osu/pull/23432 does, but somehow that pull... depends on this one?

Makes very little sense to me. Skipping review of this PR and reviewing the dependent one instead.

Reply:
Yeah, sorry about that. I'll rebase the PR chain to fix things up.